string cam_id
float64 host_timestamp
string message
---
